llvm/test/tools/yaml2obj/COFF/basic-mips.yaml

```diff
@@ -0,0 +1,96 @@
+# RUN: yaml2obj %s -o %t
+# RUN: llvm-readobj --file-headers %t | FileCheck %s
```

Comment:
I think it would be useful for the llvm-readobj output check to include the MIPS-specific bits, if that's possible?